packages/test/functional-tests/src/test/containerRuntime.spec.ts

```diff
@@ -19,7 +19,11 @@ import { IConnectionManagerFactoryArgs } from "@fluidframework/container-loader/
 // eslint-disable-next-line import/no-internal-modules
 import { ConnectionManager } from "@fluidframework/container-loader/dist/connectionManager";
 import { MockDocumentDeltaConnection, MockDocumentService } from "@fluidframework/test-loader-utils";
-import { ScheduleManager, DeltaScheduler } from "@fluidframework/container-runtime";
+// ADO:1981
```

Comment:
That's probably fine, but I'd rather see this flow using @internal tag and our tooling having appropriate handling for that (including not producing documentation, not generating .api.md files, not producing valudate*.ts files